quodlibet/po/fr.po

```diff
@@ -1413,120 +1397,112 @@ msgid ""
 msgstr ""
 
 #: ../quodlibet/ext/events/advanced_preferences.py:68
-#, fuzzy
 msgid "Advanced Preferences"
-msgstr "Préférences du navigateur à panneaux"
+msgstr "Préférences avancées"
 
 #: ../quodlibet/ext/events/advanced_preferences.py:69
 msgid "Allow to tweak advanced config settings."
 msgstr ""
 
 #: ../quodlibet/ext/events/advanced_preferences.py:156
 msgid "I know what I'm doing"
-msgstr ""
+msgstr "Je confirme comprendre ce que je fais"
 
 #: ../quodlibet/ext/events/animosd/main.py:27
 msgid "Animated On-Screen Display"
-msgstr ""
+msgstr "Incrustation d'informations"
 
 #: ../quodlibet/ext/events/animosd/main.py:28
 msgid "Displays song information on your screen when it changes."
-msgstr ""
+msgstr "Affiche des information à l'écran dès qu'un nouveau titre est lu."
```

Comment:
It displays a pop over  with track information. See screenshot. And it's everytime a track starts, so not "dès qu'il est modifié".
![qlss_animosd](https://cloud.githubusercontent.com/assets/11503399/22937887/4715f5b6-f2da-11e6-8069-b5a7bfb51678.jpg)